packages/core/src/Section/Section.tsx

```diff
@@ -77,25 +80,31 @@ export const HvSection = forwardRef<HTMLDivElement, HvSectionProps>(
         className={cx(classes.root, className)}
         {...others}
       >
-        <div className={classes.header}>
-          {expandable && (
-            <HvButton
-              icon
-              onClick={(event) => {
-                setIsOpen((o) => !o);
-                onToggle?.(event, !isOpen);
-              }}
-              aria-expanded={isOpen}
-              aria-controls={contentId}
-              aria-label={isOpen ? "Collapse" : "Expand"}
-              {...expandButtonProps}
-            >
-              {isOpen ? <Up /> : <Down />}
-            </HvButton>
-          )}
-          {title}
-          <div className={classes.actions}>{actions}</div>
-        </div>
+        {(title || actions || expandable) && (
```

Comment:
Can we add a samples of a `title`'less `HvSection` - it's really common in app mocks, and we'd cover regressions with Applitools